src/export/mod.rs

```diff
@@ -89,9 +90,18 @@ impl ExportManager {
             ExportType::Markdown => Box::new(MarkdownExporter::default()),
             ExportType::Orgmode => Box::new(OrgmodeExporter::default()),
         };
+
         self.exporters.push(ExporterWithFilename {
             exporter,
-            filename: filename.to_string(),
+            filename: if filename == "-" {
+                if env::consts::OS == "windows" {
+                    "con:".to_string()
```

Comment:
Thank you. Have you tested this on Windows and Linux? 

To distinguish between Windows and Linux, we use `cfg!(…)` elsewhere in hyperfines source code. The difference is that it is a compile-time switch. Your runtime switch might be favorable in certain (weird) scenarios, but I think I'd prefer to use `if cfg!(windows)` here instead of the `env::consts::OS` check.

Reply:
Yeah, I tested this on my linux machine and as well as my windows one. It seemed to be working